src/Utility/Workaround/LibcxxDebug.cpp

```diff
@@ -0,0 +1,22 @@
+#include <cstring>
+#include <string>
+
+#ifdef _LIBCPP_DEBUG
```

Comment:
I was under impression that this workaround required only for specific libc++ versions, mainly compiled by Apple themselves and not broad libc++ issue. If that is the case inclusion of this workaround should be limited to only affected versions, maybe via test compilation during cmake configuration?

Reply:
Yes, it's mainly an Apple issue. I think I'll just remove this one for now and then submit a better fix separately.

Reply:
Yes, that's an Apple issue. Will think of a better way to fix this.